Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -2228,4 +2284,15 @@ private extension ProductStoreTests {
         storageManager.insertSampleOrder(readOnlyOrder: orders[2]).items = NSOrderedSet(array: (0 ..< 3)
             .map { index in storageManager.insertSampleOrderItem(readOnlyOrderItem: orderItems[index]) })
     }
+
+    func prepareRecentlySoldProductSamples(with productIDs: [Int64], orderStatus: OrderStatusEnum = .completed) {
+        productIDs.forEach { storageManager.insertSampleProduct(readOnlyProduct: Product.fake().copy(productID: $0))}
+
+        let orderItems = productIDs.map { OrderItem.fake().copy(productID: $0) }
+        let orders = (0 ..< 3).map { index in Order.fake().copy(siteID: sampleSiteID, status: orderStatus, datePaid: Date().addingDays(-index)) }
+
+        storageManager.insertSampleOrder(readOnlyOrder: orders[0]).items = [storageManager.insertSampleOrderItem(readOnlyOrderItem: orderItems[0])]
+        storageManager.insertSampleOrder(readOnlyOrder: orders[1]).items = [storageManager.insertSampleOrderItem(readOnlyOrderItem: orderItems[1])]
+        storageManager.insertSampleOrder(readOnlyOrder: orders[2]).items = [storageManager.insertSampleOrderItem(readOnlyOrderItem: orderItems[2])]
```

Comment:
Nice! This can also be refactored to use `orders.enumerated()`, so we have access to the element indices:

```suggestion
for (index, _) in orders.enumerated() {
  storageManager.insertSampleOrder(readOnlyOrder: orders[index]).items = [storageManager.insertSampleOrderItem(readOnlyOrderItem: orderItems[index])]
}
```

Reply:
Thanks for the suggestion, done in 512676ff31fa86499927ff7097b3a6d1043108fe